workflows/prognostic_c48_run/runtime/loop.py

```diff
@@ -334,6 +373,19 @@ def _open_model(self, ml_config: MachineLearningConfig, step: str):
         self._log_info("Model Loaded")
         return model
 
+    def _get_radiation_wrapper(self, config: UserConfig, namelist: f90nml.Namelist):
+        if config.radiation_scheme and config.radiation_scheme.kind == "python":
+            if self.rank == 0:
+                _download_radiation_assets()
```

Comment:
Should this be in RadiationWrapper.from_config? The TimeLoop seems to have to know a lot about RadiationWrapper (methods, input data,etc). I bet you could decouple them much more.

Reply:
Yeah, it could be moved there. 

Some of this asset downloading could be avoided entirely if the radiation port could read the forcing files as the text files that are written into the run directory anyway for FV3GFS. Unfortunately the port changed its i/o to read them from netcdfs instead, so now we're effectively putting them into the run directory twice. This is an obvious todo for the port itself.